guides/common/modules/proc_importing-ostree-content.adoc

```diff
@@ -0,0 +1,73 @@
+[id="importing-ostree-content_{context}"]
+= Importing OSTree Content
+
+You can import and synchronize OSTree content from custom online sources over HTTP(S).
+
+.Prerequisite
+* A published HTTP location (upstream URL) for the OSTree to import.
+
+.Procedure
+. In the {Project} web UI, navigate to *Content* > *Products* and click *Create Product*.
+. In the *Name* field, enter a name for your OSTree content.
+This automatically populates the *Label* field.
+. Optional: From the *GPG Key* list, select the GPG key for the product.
+. Optional: From the *SSL CA Cert* list, select the SSL CA certificate for the product.
+. Optional: From the *SSL Client Cert* list, select the SSL client certificate for the product.
+. Optional: From the *SSL Client Key* list, select the SSL client key for the product.
+. Optional: From the *Sync Plan* list, select an existing sync plan or click *Create Sync Plan* and create a sync plan for your product requirements.
+. In the *Description* field, enter a description of the product.
+. Click *Save*.
+. When the product creation completes, click *New Repository*.
+. In the *Name* field, enter a name for the repository.
+This automatically populates the *Label* field.
+. From the *Type* list, select `ostree`.
+. In the *Upstream URL* field, enter the URL of the external repository to use as a source.
+For example `http://www.example.com/rpm-ostree/`.
```

Comment:
```suggestion
For example `\http://www.example.com/rpm-ostree/`.
```

Maybe we should escape the link? example.com/anything should never be clicked/clickable.